api/src/test/resources/registration_app.json

```diff
@@ -140,35 +140,6 @@
               ]
             }
           ]
-        },
-        {
-          "id": "ethnicity",
-          "label": "Ethnicity",
-          "questions": [
-            {
-              "legend": "acme.app.registerPatient.section.ethnicity",
-              "fields": [
-                {
-                  "type": "obs",
-                  "formFieldName": "obs.ACME:912",
-                  "label": "acme.app.registerPatient.ethnicity",
```

Comment:
Replace this with `"label": "Heart rate"`.

Reply:
Changed and also updated the other section labels.